tools/cli/NearObjectCli.cxx

```diff
@@ -227,12 +227,25 @@ NearObjectCli::AddSubcommandUwbRangeStart(CLI::App* parent)
 
     // TODO is there a way to put all the enums into a list of [optionName, optionDestination, optionMap] so we don't have to create the initializer list each time
 
-    // enumerations
+    // List mandatory params first
+    // TODO: --help should output that these params are mandatory
     detail::AddEnumOption(rangeStartApp, uwbConfig.deviceRole);
+    detail::AddEnumOption(rangeStartApp, uwbConfig.multiNodeMode);
+    rangeStartApp->add_option("--NumberOfControlees", uwbConfig.numberOfControlees, "1 <= N <= 8")->capture_default_str(); // TODO: Input validation
```

Comment:
You can indicate that a param is mandatory on the help text by doing something like
```
rangeStartApp->add_option("--NumberOfControlees", uwbConfig.numberOfControlees, "1 <= N <= 8   This parameter is mandatory for start ranging")->capture_default_str();
```

Reply:
Oh yeah that's true. It'll take an extra step to do something similar for the enums using AddEnumOption, but I can make it work